src/pages/brain/[pageId].tsx

```diff
@@ -0,0 +1,240 @@
+import {
+  Button,
+  Img,
+  Link,
+  Tag,
+  TagLabel,
+  TagLeftIcon,
+  Wrap,
+} from "@chakra-ui/react"
+import BrainCard from "components/brain/BrainCard"
+import Layout from "components/common/Layout"
+import LinkPreviewHead from "components/common/LinkPreviewHead"
+import Section from "components/common/Section"
+import CategorySection from "components/explorer/CategorySection"
+import { GetStaticPaths } from "next"
+import { NotionAPI } from "notion-client"
+import { DiscordLogo, Globe, IconProps, TwitterLogo } from "phosphor-react"
+
+import { PropsWithChildren } from "react"
+import { NotionRenderer } from "react-notion-x"
+import "react-notion-x/src/styles.css"
+import { BrainCardData } from "types"
+
+type CustomPageLinkProps = {
+  href: string
+  children: any
```

Comment:
should wrap the type with PropsWithChildren instead of specifying own children prop, which is already done at line 37 (I don't see CustomPageLinkProps being used anywhere else) so I don't get why it's here